pkg/commands/copy.go

```diff
@@ -126,6 +120,21 @@ func (c *CopyCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.Bu
 	return nil
 }
 
+func getUserGroup(chownStr string, env []string) (int64, int64, error) {
+	if chownStr == "" {
+		return util.DoNotChangeUID, util.DoNotChangeGID, nil
+	}
+	chown, err := util.ResolveEnvironmentReplacement(chownStr, env, false)
+	if err != nil {
+		return -1, -1, err
```

Comment:
Why -1 and not `util.DoNotChangeUID` ?

Reply:
The return value here should be ignored since `err` is not nil. 
I did not use the `util.DoNotChangeUID` since the return value does not matter.